### PR TITLE
Feature CoffeeCommand is implemented 

### PR DESCRIPTION
**Pull request by dkoshy**
Coffee commands and command handler is implemented

### CODE REVIEW

File: SampleConsoleSolutions/WirebrainCoffeeShop.StockInfoApp/CoffeeCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            if (stockinfo?.Count == 0)
+            {
+                Console.WriteLine("Command not Found Please try again !!!");
+            }
+
+            else if (stockinfo.Count == 1)
+            {
+                var stock = stockinfo.FirstOrDefault();
+                Console.WriteLine($"Coffee shope Location is : {stock?.Location}  ");
+                Console.WriteLine($"Coffee shope Location is : {stock?.BeanStockinKg}  ");
```
Comment (binikoshy) [inline SampleConsoleSolutions/WirebrainCoffeeShop.StockInfoApp/CoffeeCommandHandler.cs:34]: Hi,  @dkoshy   Pleas add coffee cup stock details also.

File: SampleConsoleSolutions/WirebrainCoffeeShop.StockInfoApp/CoffeeCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            {
+                Console.WriteLine("Command not Found Please try again !!!");
+            }
+
+            else if (stockinfo.Count == 1)
+            {
+                var stock = stockinfo.FirstOrDefault();
+                Console.WriteLine($"Coffee shope Location is : {stock?.Location}  ");
+                Console.WriteLine($"Coffee shope Beanstock is : {stock?.BeanStockinKg}  ");
+                Console.WriteLine($"Coffee shope Paper cup stock is : {stock?.CoffeeCupStock}  ");
```
Comment (dkoshy) [inline SampleConsoleSolutions/WirebrainCoffeeShop.StockInfoApp/CoffeeCommandHandler.cs:35]: Hey @binikoshy  I added stock details

File: SampleConsoleSolutions/WirebrainCoffeeShop.StockInfoApp/CoffeeCommandHandler.cs (hunk 1 above)
Comment (binikoshy) [reply SampleConsoleSolutions/WirebrainCoffeeShop.StockInfoApp/CoffeeCommandHandler.cs]: Thank you

**Comment by dkoshy** [comment]
Merge "feature/CoffeeCommandImplimentation"   changes into master branch

**Pull request merged**